Week3/StringSeg.java

```diff
@@ -0,0 +1,68 @@
+import java.util.HashSet;
+import java.util.Set;
+
+public class StringSeg {
+
+	public static boolean breakable(String line, Set<String> Dictionary) {
+		
+		if(Dictionary== null && (line==null || line=="")) 
+		{
+			return true;

```

Comment:
I'm not sure about this case.  This would be a good question to clarify with your instructor, but I'm not sure that this case makes much sense.  The way you have it (Dictionary=null, line = null) returns true, but (Dictionary=empty, line = null/empty) returns false.

Presumably you're trying to handle the case empty string is a subset of empty set, which I would assume would be ((Dictionary == null || Dictionary.isEmpty()) && (line == null || line == "")) => true

Reply:
Yes I discussed all cases on slack channel. I got the sense. I will update code